lib/puppet/provider/f5_license/rest.rb

```diff
@@ -3,30 +3,9 @@
 
 Puppet::Type.type(:f5_license).provide(:rest, parent: Puppet::Provider::F5) do
 
-  def self.instances
```

Comment:
`@property_hash` is populated by `instances`.  Since it is now always an empty hash, `exists?` will always return false.